.github/workflows/pr-commands.yaml

```diff
@@ -53,9 +53,9 @@ jobs:
       - uses: r-lib/actions/pr-push@v1
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-  style-stan:
-    if: startsWith(github.event.comment.body, '/style-stan')
-    name: style-stan
+  stan:
```

Comment:
I'd name it `format` instead of `stan` as it can be used for C/C++ code formatting in any project.

Reply:
Also, we may use `ubuntu-20.04` instead of `ubuntu-16.04` in all workflows.

Reply:
I updated this to cpp on master (needed to do these changes on master in order to test the GitHub Actions (I am not aware of a better way to test GAs))

Reply:
Why did you add it to `pr-commands.yaml` workflow? I'd test with `on push` events for easy and quick triggers. I'm testing a `Lint-Format.yaml` workflow [here](https://github.com/hsbadr/HiClimR).